### PR TITLE
feat: Implement WebSocket Notifications for Ticket Review Updates

### PR DESCRIPTION
**Pull request by saithsab877**
## Type of change

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Checklist before requesting a review
- [ ] I have performed a self-review of my code
- [ ] I have tested on Chrome and Firefox
- [ ] I have tested on a mobile device
- [ ] I have provided a screenshot or recording of changes in my PR if there were updates to the frontend

### CODE REVIEW

**Comment by saithsab877** [comment]
Hi @humansinstitute Please review this PR.

File: handlers/ticket.go [hunk 1]
```diff
@@ -484,6 +484,29 @@ func (th *ticketHandler) ProcessTicketReview(w http.ResponseWriter, r *http.Requ
 		return
 	}
 
+	ticketMsg := websocket.TicketMessage{
+		BroadcastType:   "direct",
+		SourceSessionID: reviewReq.RequestUUID,
```
Comment (humansinstitute) [inline handlers/ticket.go:489]: Can we change this. This should be reviewReq.sourceWebsocket

**Comment by humansinstitute** [comment]
Once that is changed looks good

**Comment by saithsab877** [comment]
Hi @humansinstitute Fixed.

**Pull request merged**